lib/modules/txt.mjs

```diff
@@ -520,6 +520,48 @@ export async function validateDocumentName (doc, { mode = MODES.NORMAL } = {}) {
   return result
 }
 
+/**
+ * The 'RFC 5378 fix' announced around 5 Feb 2009 means that drafts first submitted before 10 Nov 2008 with a high probability should have an additional boilerplate paragraph excempting it from full 5378 compliance.
+ * On the other hand, a draft first submitted on or after 10 Nov 2008 should probably not have such a paragraph.
+ *
+ * @param {Object} doc Document to validate
+ * @param {Object} [opts] Additional options
+ * @param {number} [opts.mode=0] Validation mode to use
+ * @returns {Array} List of errors/warnings/comments or empty if fully valid
+ */
+export async function validatePre5378Documents (doc, { mode = MODES.NORMAL } = {}) {
+  const result = []
+  const rfcDocumentNumber = 5378
+
+  const updatesAndObsoletesRfc = [...doc.data.extractedElements.updatesRfc, ...doc.data.extractedElements.obsoletesRfc]
+
+  if (!updatesAndObsoletesRfc.length) return result
+
+  const invalidRfcNumbers = updatesAndObsoletesRfc.filter(num => num >= rfcDocumentNumber)
```

Comment:
you are comparing array of strings to array of numbers. And also for valid comparation we should use num <= rfcDocumentNumber, if we wanna find invalid rfcs